### DISCUSSION TITLE
Laravel Nova Log Viewer

### DISCUSSION THREAD

**Discussion by clementmas** [opcodesio/log-viewer#7] [Ideas]
Is there a Laravel Nova plugin for this Log Viewer? If not, is that something planned?

Last time I checked the log viewer packages available for Nova were not very good.

Thanks!

**Comment by arukompas** [comment]
Great idea @clementmas ! Not exactly sure whether Log Viewer would be compatible with Nova, since Log Viewer uses Livewire/Alpine, and Nova is built with Vue.

Nevertheless, a quick workaround is to add a link to the log viewer in the Nova menu bar. And then configure Log Viewer's "back link" to point to the Nova dashboard. Easy navigation! 😄

**Reply by chrillep** [reply]
here's the Nova version for ref: https://novapackages.com/packages/laravel/nova-log-viewer

not as good looking. but I do like having it in the same interface

**Reply by chrillep** [reply]
until then something like this should do it. just to adds the link in nova 3 sidebar :) 

```blade
@env('local')
    @if (Route::has('blv.index'))
        <h3 dusk="logs" class="flex items-center font-normal text-white mb-6 text-base no-underline">
            <svg viewBox="0 0 24 24" class="sidebar-icon">
                <path fill="var(--sidebar-icon)" d="M20.59 12l-3.3-3.3a1 1 0 1 1 1.42-1.4l4 4a1 1 0 0 1 0 1.4l-4 4a1 1 0 0 1-1.42-1.4l3.3-3.3zM3.4 12l3.3 3.3a1 1 0 0 1-1.42 1.4l-4-4a1 1 0 0 1 0-1.4l4-4a1 1 0 0 1 1.42 1.4L3.4 12zm7.56 8.24a1 1 0 0 1-1.94-.48l4-16a1 1 0 1 1 1.94.48l-4 16z"/>
            </svg>
            <a href="{{ route('blv.index') }}" class="text-white no-underline dim">Logs</a>
        </h3>
    @endif
@endenv
 ```
